consensus/types/src/fork_context.rs

```diff
@@ -1,15 +1,30 @@
 use parking_lot::RwLock;
 
 use crate::{ChainSpec, Epoch, EthSpec, ForkName, Hash256, Slot};
-use std::collections::{ HashMap, HashSet};
+use std::collections::BTreeMap;
+
+#[derive(Debug, Clone)]
+pub struct HardFork {
```

Comment:
This also includes BPO forks though right? 
The naming seems to imply it excludes them. We could just make this private and call it `Fork`